src/ghost/ghost-post.service.integration.test.ts

```diff
@@ -507,27 +473,6 @@ describe('GhostPostService', () => {
             expect(Post.isDeleted(deletedPost!)).toBe(true);
         });
 
-        it('should return an error when deletion fails', async () => {
```

Comment:
Same for this. This test doesn't have much to be an integration test. Makes more sense to be tested [here](https://github.com/TryGhost/ActivityPub/pull/1073/files#diff-5c81e24007c2cfda5c5b7b00fc25c87538a30c855661cbe153797a7b8cc6c4a2R279)